### ISSUE TITLE
Fluxstore iOS App Rejected on Appstore 

### ISSUE THREAD

**Issue by jeeali** [inspireui/support#3448]
Step 1 (require): let us know your detail issues & screenshots, what is the product name and version?

#3376 
Here is the main issue link: https://github.com/inspireui/support/issues/3376
It looks like, I can't reopen that issue if a collaborator closed it. So, I'm creating a new issue.

Reply to your last comment 
( https://github.com/inspireui/support/issues/3376#issuecomment-633346504_ )

### 1. Below are the Appstore Description, Banner Intro. Please check.

**Promotional Text:** Need your fashion fix on the move? Shop using the Ciera Couture app

**Description:** Ciera Couture is an affordable online shopping platform with a distinct tone focusing on women's fashion. No matter what the current fashion trend is, CieraCouture.com is sure to follow it or even lead it. With all the categories like women's clothing, shoes and accessories. Ciera Couture is a one-stop destination for fashionistas. Shopping becomes even more enjoyable with the new Ciera Couture mobile app.

• Buy straight from our app to get essential new season women's clothing delivered straight to your door.
• Don't have time right now? No worries, add your favorite outfit to your cart, add to your wishlist to add to bag later.
• While you're on your phone - join our squad and share your selfie with the hashtag #3174

**Keywords:** ciera couture, fashion, women, online shopping

**2. Current Payment is Paypal only.**

**3. Categories**
Main Category-> Shopping
Sub Category -> Lifestyle

**4. I added you as a Tester in TestFlight**

------

Step 2 (require): verify license and sharing config files via this google form - http://verify.inspireui.com (please don't share the screenshot purchase here)

I filled the form as well

*Important Note:* 
- Please help to fill the required information from the above 2 steps and don't share the public screenshot purchase here, we may close the ticket which is not included the proof of purchase. Kindly help us to create only 1-Ticket & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but would like to create a new ticket, then you don't need to submit the form again, just do simple step by linking to the previous verified ID, example: #Ticket-ID
Thank you so much for your time 😊

**Comment by minhcasi** [comment]
We will check your Testfligh and give some feedback - thank you!

**Comment by jeeali** [comment]
I changed keyword, description, and talk with the Appstore team. They Approved it.

**Comment by minhcasi** [comment]
Congratulation, could you share your app URL once it is ready on Appstore - thank you!

**Comment by jeeali** [comment]
Thank you!
Here is the link: https://apps.apple.com/us/app/ciera-couture/id1514711705
